tests/end_to_end/test_linea_api.py

```diff
@@ -73,15 +94,21 @@ def test_save_different_session(execute):
     )
 
 
-def test_save_twice(execute):
+@pytest.mark.parametrize(
+    "move_folder",
+    [options.safe_get("artifact_storage_dir")],
+    indirect=True,
+)
+def test_save_twice(execute, move_folder):
```

Comment:
what is this for and why isnt it used? if its going to be used before every test, can you make it into a fixture that is autoused?

Reply:
I changed the tests to use a mark to pass data instead of parametrization. Is it fine now or still too much repeated code?

Reply:
look mostly good. try the autouse fixture sample and see if it helps futher. otherwise good to go.

```
@pytest.fixture(autouse=True)
@pytest.mark.folder(options.safe_get("artifact_storage_dir"))
def test_save_different_session(move_folder):
    yield
```